modules/shredder/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/shredder/batch/spark/TypesAccumulator.scala

```diff
@@ -20,23 +20,23 @@ import com.snowplowanalytics.iglu.core.SchemaKey
 
 import com.snowplowanalytics.snowplow.analytics.scalasdk.Data
 
-import com.snowplowanalytics.snowplow.rdbloader.common.LoaderMessage.{Format, ShreddedType}
-import com.snowplowanalytics.snowplow.rdbloader.shredder.batch.spark.ShreddedTypesAccumulator._
+import com.snowplowanalytics.snowplow.rdbloader.common.LoaderMessage.{TypesInfo, SnowplowEntity}
+import com.snowplowanalytics.snowplow.rdbloader.shredder.batch.spark.TypesAccumulator._
 
-class ShreddedTypesAccumulator extends AccumulatorV2[KeyAccum, KeyAccum] {
+class TypesAccumulator[T] extends AccumulatorV2[KeyAccum[T], KeyAccum[T]] {
```

Comment:
This class could use a scaladoc, especially since now it carries a type parameter.